orderbook/src/main.rs

```diff
@@ -92,11 +94,18 @@ async fn main() {
     )
     .await
     .expect("failed to create gas price estimator");
-    let price_estimator = UniswapPriceEstimator::new(Box::new(PoolFetcher {
-        factory: uniswap_factory,
-        web3,
-        chain_id,
-    }));
+
+    let mut base_tokens = HashSet::from_iter(args.shared.base_tokens);
```

Comment:
So base tokens are something we specify as a runtime argument? What types of base tokens do we have in mind? Is this just a short list of reputable tokens or some very long list of everything we consider potentially tradable?

Reply:
Base tokens are copied from here: https://github.com/gnosis/gp-v2-swap/blob/a62ded4838775c8c53f934cd3d957e3704fe85ad/src/constants/index.ts#L66-L70

Yes, it's just a list of most liquid tokens for each network.